specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2018-03-01-preview/examples/createBasicAKSCompute.json

```diff
@@ -13,18 +13,6 @@
         }
     },
     "responses": {
-        "200": {
```

Comment:
I see there are differences some files checked in to MLCompute repo and here. Can you make sure that both place we have the same swagger files?

Reply:
yes, after I get this checked in the azure repo, I will backport it to our repo.
But, for the future, I think that we should not maintain our own swagger, and everything should be changed here.
There is not sense in maintaining a swagger in both places